### PR TITLE
Panorama web search feature improvements

### PR DESCRIPTION
**Pull request by ankurjuneja**
#### Rationale
This is the continued work for the related PR addressing manual testing issues and feature improvements.

#### Related Pull Requests
* https://github.com/LabKey/MacCossLabModules/pull/243

#### Changes
* improved UI for search
* search substring of peptides and proteins in the whole sequence instead of prefix string search
* fix containerfilter parse error

### CODE REVIEW

File: panoramapublic/src/org/labkey/panoramapublic/view/search/panoramaWebSearch.jsp [hunk 1]
```diff
@@ -349,25 +366,38 @@
...
                                         itemId: proteinNameItemId,
-                                        labelCls: 'labkey-form-label',
+                                        labelAlign: 'top',
                                         labelWidth: 125,
                                         listeners: {
                                             render: function (comp, eOpts) {
                                                 checkAndFillValuesFromUrl(proteinNameItemId, comp);
+                                                new Ext4.ToolTip({
+                                                    target : comp.getEl(),
+                                                    html: 'Required to search for proteins. You may use the name as specified by the FASTA file, or an annotation, such as a gene name, that has been loaded from an annotations file. You may comma separate multiple names.'
```
Comment (labkey-jeckels) [inline panoramapublic/src/org/labkey/panoramapublic/view/search/panoramaWebSearch.jsp]: Does comma separation actually work here? Wouldn't it just be searching for that exact text?

File: panoramapublic/src/org/labkey/panoramapublic/view/search/panoramaWebSearch.jsp [hunk 2]
```diff
@@ -376,33 +406,47 @@
...
                                         itemId: peptideSequenceItemId,
-                                        labelCls: 'labkey-form-label',
+                                        labelAlign: 'top',
                                         labelWidth: 125,
                                         listeners: {
                                             render: function (comp, eOpts) {
                                                 checkAndFillValuesFromUrl(peptideSequenceItemId, comp);
+                                                new Ext4.ToolTip({
+                                                    target : comp.getEl(),
+                                                    html: 'Enter the peptide sequence to find, or multiple sequences separated by commas. Use * to match any sequence of characters.'
```
Comment (labkey-jeckels) [inline panoramapublic/src/org/labkey/panoramapublic/view/search/panoramaWebSearch.jsp]: Same thing - does this work?

File: panoramapublic/src/org/labkey/panoramapublic/view/search/panoramaWebSearch.jsp [hunk 3]
```diff
@@ -376,33 +406,47 @@
...
-                                        labelCls: 'labkey-form-label',
                                         input: true,
-                                        labelWidth: 125,
+                                        labelWidth: 130,
                                         listeners: {
                                             render: function (comp, eOpts) {
                                                 checkAndFillValuesFromUrl(exactPeptideMatchesItemId, comp);
+                                                new Ext4.ToolTip({
+                                                    target : comp.getEl(),
+                                                    html: 'If checked, the search will match the peptides exactly; if unchecked, it will match any peptide that contain the specified sequence and ignore modifications.'
```
Comment (labkey-jeckels) [inline panoramapublic/src/org/labkey/panoramapublic/view/search/panoramaWebSearch.jsp]: This seems suspect too - where are we switching between searching the modified and unmodified forms?

**Review by labkey-jeckels — CHANGES_REQUESTED** [review]

File: panoramapublic/src/org/labkey/panoramapublic/view/search/panoramaWebSearch.jsp (hunk 1 above)
Comment (ankurjuneja) [reply panoramapublic/src/org/labkey/panoramapublic/view/search/panoramaWebSearch.jsp]: it doesn't, these tips are copy paste texts. thanks for catching them.

**Review by labkey-jeckels — APPROVED** [review]

**Pull request merged**